falco/assets/logs/falco_tests.yaml

```diff
@@ -0,0 +1,151 @@
+id: "falco"
+tests:
+ -
+  sample: |-
+    {
+      "output" : "09:48:02.285310579: Warning Sensitive file opened for reading by non-trusted program (file=/etc/shadow gparent=containerd-shim ggparent=systemd gggparent=<NA> evt_type=openat user=root user_uid=0 user_loginuid=-1 process=cat proc_exepath=/usr/bin/cat parent=runc command=cat /etc/shadow terminal=0 container_id=c1f3a8646e7f container_image=docker.io/library/nginx container_image_tag=latest container_name=nginx k8s_ns=default k8s_pod_name=nginx-7854ff8877-vrlrs)",
+      "output_fields" : {
+        "container" : {
+          "image" : {
+            "tag" : "latest",
+            "repository" : "docker.io/library/nginx"
+          },
+          "name" : "nginx",
+          "id" : "c1f3a8646e7f"
+        },
+        "evt" : {
+          "time" : 1730281682285310500,
+          "type" : "openat"
```

Comment:
Something you could consider is remapping this to the `Events | evt.name` [standard attribute](https://docs.datadoghq.com/standard-attributes/?product=log+management&search=evt.name) and corresponding facet, if that makes sense in your context.

Reply:
We have remapped it with `evt.name` standard attribute.